docs/source/tutorial_crowdsourcing.md

```diff
@@ -0,0 +1,255 @@
+
+- `conf/example.yaml`: the file of Hydra parameter values that are set by your task by default when lauching `run.py`. The parameter values in this file should be set so as to easily demonstrate the basic functionality of your task without requiring additional configuration steps.
+- `task_config/`: the standard folder in which useful configuration files are stored, such as for specifying UI text, configuring models, providing sample onboarding parameters, etc.
+
+A few things to keep in mind:
+
+1.  To end a conversation, you should check to see if an action has
+    `'episode_done'` set to `True`, as this signals that the world should
+    start returning `True` for the `episode_done()` function.
+2.  Make sure to test your dialog task using Mephisto's sandbox mode (enabled by default) before
+    pushing it live. See the [crowdsourcing README](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing#running-tasks-live) for running live tasks.
+    
+Advanced Task Techniques
+------------------------
+
+The Mephisto platform allows for a number of advanced customization
+techniques to cover specialized tasks. See the [`bootstrap-chat` README](https://github.com/facebookresearch/Mephisto/blob/master/packages/bootstrap-chat/README.md) for a discussion of how to utilize Bootstrap-based UI components for crowdsourcing tasks.
+
+Running a Task
+--------------
+
+If you have not used Mechanical Turk before, you will need an MTurk
+Requester Account and an AWS account (these are two separate accounts).
+Follow the steps below:
+
+-   Sign up for an AWS account at
+    [aws.amazon.com](https://aws.amazon.com/).
+-   Sign up for an MTurk account at
+    [requester.mturk.com](https://requester.mturk.com/).
+-   Go to the developer tab
+    ([<https://requester.mturk.com/developer>](https://requester.mturk.com/developer))
+    and link your AWS account to your MTurk account (Step 2 on that
+    screen).
+-   MTurk also has a “Sandbox”, which is a test version of the MTurk
+    marketplace. You can use it to test publishing and completing tasks
+    without paying any money. ParlAI supports the Sandbox. To use it, you will need to sign up for a [Sandbox
+    account](http://requestersandbox.mturk.com/), and you will then also
+    need to [link your AWS
+    account](http://requestersandbox.mturk.com/developer) to your
+    Sandbox account. In order to test faster, you will also want to
+    create a [Sandbox Worker account](http://workersandbox.mturk.com/).
+    You can then view tasks that you publish from ParlAI and complete them
+    yourself.
+-   Mephisto's default MTurk functionality requires a free Heroku account,
+    which can be obtained [here](https://signup.heroku.com/). Running
+    any Mephisto MTurk operation will walk you through linking the two.
+    
+To run a crowdsourcing task, launch its run file (typically `run.py`) with the proper flags, using a command like the following:
```

Comment:
I think it would be worth it to include the argument specifying which `config.yaml` file is being used; I know that that is in the crowdsourcing README but I think it's a pretty important feature

Reply:
(see response below)